Source/NSubstitute/Exceptions/CouldNotSetReturnException.cs

```diff
@@ -45,6 +45,17 @@ public class CouldNotSetReturnDueToNoLastCallException : CouldNotSetReturnExcept
 #if NET35 || NET4 || NET45
     [Serializable]
 #endif
+    public class CouldNotSetReturnDueToMissingInfoAboutPreviousCallException : CouldNotSetReturnException
+    {
+        public CouldNotSetReturnDueToMissingInfoAboutPreviousCallException() : base("Could not find information about the previous call to return from.") { }
```

Comment:
Ah, that `CouldNotSetReturnDueToMissingInfoAboutPreviousCallException` class extends `CouldNotSetReturnException` so the standard info will present there. My bad.
I'm OK to have a separate exception type but just think that it's useless for end user. As for NSub developer then it's easy for find out the source of it in case of repro.

One thing I would change for sure is "previous" to "last". We call it "last" call everywhere, so that it will be consistent. There are more lines with "previous": https://github.com/nsubstitute/NSubstitute/pull/264/files#diff-eddc8d190e75c5fd320ea3c3c777480bR25